_posts/0000-01-02-Yesman-coder.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
👍🏾 🥇 I know it does not look like that at the moment, but we will make it I promise. 🥇 👍🏾